src/page.rs

```diff
@@ -261,8 +261,10 @@ impl Page {
         Ok(resp.result.node)
     }
 
-    pub async fn close(self) {
-        todo!()
+    /// Closes page
+    pub async fn close(self) -> Result<()> {
+        self.execute(CloseParams {}).await?;
```

Comment:
This is the correct command, but it should be changed to `CloseParams::default()` to follow the convention used for empty commands in this project.